far/stddlg.cpp

```diff
@@ -348,41 +354,48 @@ int GetSearchReplaceString(
 
 	Dlg->Process();
 
-	if (const auto ExitCode = Dlg->GetExitCode(); ExitCode == dlg_button_action || ExitCode == dlg_button_all)
+	const auto ExitCode = Dlg->GetExitCode();
+
+	if (none_of(ExitCode, dlg_button_action, dlg_button_up, dlg_button_all))
```

Comment:
The conventional pattern is `if (ExitCode < 0)` - no point in mentioning all the buttons twice here and below.

Reply:
But the `Cancel` button has a positive code. And I do not want to mention it here explicitly. Are there any other ways to cancel a dialog? My variant is safe. Let's keep it.

Reply:
> But the `Cancel` button has a positive code

Thanks, it's `exitcode == cancel or exitcode < 0` of course.

I mean if we add a new button here one day, it will need to go into two places instead of one.
But anyway, it's minor.

Reply:
OK. Done.